src/collections/posts/hidden-features-of-beatsaver.md

```diff
@@ -0,0 +1,91 @@
+
+![](/uploads/posts/uncovering-beatsaber-features-3.png)
+
+The results from the image above will now show you maps that either have the "Fitness" or "Tech" or both tags. If you were trying to find maps that are techy Fitbeat maps you cannot find them this way though.
+
+However, you can combine it with the tag exclusion feature and find your techy Fitbeat maps by excluding every tag that is not "Tech" and "Fitness" by pressing `[Shift]` and `[Alt]` together while clicking the tags.
+
+![](/uploads/posts/uncovering-beatsaber-features-4.png)
+
+This works because multiple negative filters will - other than the positive filters - result in being interpreted as "Neither X, Y, nor Z or any combination of those".
+
+# Multi-Adding to Bookmarks
+
+On playlists you have a useful "Multi-Add" button that you can use to add multiple maps by their key or hash.
+
+![](/uploads/posts/uncovering-beatsaber-features-5.png)
+
+When it comes to your bookmarks, you will not find buttons like that.
+
+![](/uploads/posts/uncovering-beatsaber-features-6.png)
+
+Do not let that fool you! Just add a `/add` to the end of your bookmark URL and you can still multi-add! For example, `https://beatsaver.com/playlists/97886/add` (this link will not work for you though).
+
+You can also add `/edit` to the URL, but this will not do a lot for you since the API will not allow editing the playlist.
+
+# "key:" keyword in search
+
+If you know the key of a map, you can navigate to the map page directly by typing
+
+`key:{your map key}`
+
+in the search bar and then clicking "Search".
+
+![](/uploads/posts/uncovering-beatsaber-features-7.png)
+
+# No Double Upload
+
+When you try to upload a map that has already been uploaded it will not allow you to do so and tell you that the Map has already been uploaded. This is checked via the map hash. The upload will also generally verify, that your map file is valid and will work in-game.
+
+![](/uploads/posts/uncovering-beatsaber-features-8.png)
+
+# Review Curations
+
+You may be aware that maps can be curated by our curators. However, Reviews can also be curated. This is sometimes also being referred to as being "featured". Curators and Admins can click a button to feature a given review. The review will then be shown above all uncurated reviews and have a green border. There is by the way no notification for getting your review curated.
+
+![](/uploads/posts/uncovering-beatsaber-features-9.png)
+
+# Showing Bot Maps
+
+If you ever wondered where all the bot maps are that people talk about, then you may not be aware of the fact, that BeatSaver - by default - hides bot maps. If you want to see bot maps as well, you will need to enable that in the filters!
```

Comment:
If you ever wondered where all the bot maps are that people talk about, then you may not be aware of the fact that BeatSaver, by default, hides bot maps.